EC-Switch-Seclog.py

```diff
@@ -383,6 +388,16 @@ def get_account_id(session):
         else:
             raise error
 
+def pre_flight_cfn(session):
+    status = True
+    cfn = session.client('cloudformation')
+    response = cfn.describe_stacks()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `describe_stacks` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `IsTruncated`, `NextToken`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)